packages/chat/src/components/AgentMessage/CopyButton/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { copyButton } from './CopyButton.css';
+
+export const CopyButton = ({ value = '' }: { value: React.ReactNode }) => {
+  const handleClick = () => {
+    if (value) {
+      navigator.clipboard.writeText(value.toString()).catch((err) => {
+        console.error('Failed to copy text: ', err);
+      });
+    }
+  };
+
+  return (
+    <button onClick={handleClick} className={copyButton}>
+      <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+        <path
+          fillRule="evenodd"
```

Comment:
can we import this as an SVG asset (like the others) rather than inlining it in a component?

Reply:
@mikaalnaik If you want to extract this later, then I modified the `Icon` component [here](https://github.com/voiceflow/react-chat/pull/215), so we can use it with that.